src/Azure/Orleans.Clustering.CosmosDB/Models/SiloEntity.cs

```diff
@@ -0,0 +1,40 @@
+namespace Orleans.Clustering.CosmosDB.Models;
+
+internal class SiloEntity : BaseClusterEntity
+{
+    public override string EntityType => nameof(SiloEntity);
+
+    [JsonPropertyName(nameof(Address))]
+    public string Address { get; set; } = default!;
+
+    [JsonPropertyName(nameof(Port))]
+    public int Port { get; set; }
+
+    [JsonPropertyName(nameof(Generation))]
+    public int Generation { get; set; }
+
+    [JsonPropertyName(nameof(Hostname))]
+    public string Hostname { get; set; } = default!;
+
+    [JsonPropertyName(nameof(Status))]
+    public int Status { get; set; }
+
+    [JsonPropertyName(nameof(ProxyPort))]
+    public int? ProxyPort { get; set; }
+
+    [JsonPropertyName(nameof(SiloName))]
+    public string SiloName { get; set; } = default!;
+
+    [JsonPropertyName(nameof(SuspectingSilos))]
+    public List<string> SuspectingSilos { get; set; } = new();
+
+    [JsonPropertyName(nameof(SuspectingTimes))]
+    public List<string> SuspectingTimes { get; set; } = new();
+
+    [JsonPropertyName(nameof(StartTime))]
+    public DateTimeOffset StartTime { get; set; }
+
+    [JsonPropertyName(nameof(IAmAliveTime))]
+    public DateTimeOffset IAmAliveTime { get; set; }
```

Comment:
Is there any guidance on camel vs pascal casing for JSON within the Orleans eco system?

@ReubenBond

Reply:
There's no guidance, since Orleans at its core doesn't rely on JSON. Providers should do whatever is idiomatic for the technology they target (Cosmos DB in this case). If there are no particular idioms, then they should make a choice and try to remain consistent to it.

Reply:
I'm setting them explicit here to avoid breakages when someone simply rename the property with refactory feature on VS. The default, without the attribute, is camelCase tho.

Reply:
I think Cosmos tends to default to camelCase in most examples I've seen, I think if you remove the attribute and set the serialiser setting to that as a default it should do the trick.